lib/cog/request.rb

```diff
@@ -21,7 +21,16 @@ def populate_options
       return {} if ENV['COG_OPTS'].nil?
 
       options = ENV["COG_OPTS"].split(",")
-      Hash[options.map { |opt| [ opt, ENV["COG_OPT_#{opt.upcase}"] ]}]
+      Hash[options.map { |opt| [ opt, opt_val(opt) ] }]
+    end
+
+    def opt_val(opt)
```

Comment:
Some documentation about what all this is supposed to be doing would be good, too.